vzstorage-pd/vendor/github.com/virtuozzo/goploop-cli/ploopvol/ploop-volume.go

```diff
@@ -0,0 +1,65 @@
+package ploopvol
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+)
+
+var (
+	ploopVolumeBinary = "/usr/sbin/ploop-volume"
+)
+
+type PloopVolume struct {
+	Path string
+}
+
+func (pv *PloopVolume) Create(size uint64, image string) error {
```

Comment:
I think it is better to use the same interface with ploop.Create(). The library has to follow the same principals for all functions.